tests/client.ts

```diff
@@ -1,13 +1,15 @@
+// @ts-ignore
```

Comment:
1. Let's create a `.d.ts` file so this import would work in a typesafe way.
2. Don't use `ts-ignore` - `ts-expect-error` is strictly superior.